docs/blog/new-community-coc.rst

```diff
@@ -0,0 +1,19 @@
+.. post:: Feb 20, 2017
+   :tags: community, code of conduct, announcements
+
+New and Improved Community Code of Conduct
+==========================================
+
+Greetings documentarians! As you know, Write the Docs is an incredibly diverse, friendly, and inclusive community, and we are determined to diligently continue to encourage and support safe spaces in our events and online platforms.
```

Comment:
"determined to diligently continue to encourage" is a mouthful. Maybe:

Greetings documentarians! As you know, Write the Docs is an incredibly diverse, friendly, and inclusive community. We continue to encourage and support safe spaces in our events and online platforms.